continuous_load_testing/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM golang
 COPY continuous_load_testing /app/bin/continuous_load_testing
 # Uncomment the line below to enable CloudPath instead of DirectPath
-# ENV GOOGLE_CLOUD_DISABLE_DIRECT_PATH=true
+# ENV DISABLE_DIRECT_PATH=true
```

Comment:
We don't need a new way to config the binary. Please add a flag to enable/disable directpath, just like the other flags we already have (--methods, --concurrency, etc).